introduction.md

```diff
@@ -0,0 +1 @@
+Hello, my name is Dan! I am studying software development at Bloc, and part of my coursework is to complete CodeSchool's GitHub course. Nice to meet you!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.